src/main/io/gps.c

```diff
@@ -522,6 +607,48 @@ static void ubloxSendNAV5Message(uint8_t model)
     ubloxSendConfigMessage(&tx_buffer, MSG_CFG_NAV_SETTINGS, sizeof(ubxCfgNav5_t));
 }
 
+static void ubloxSendNav5XMessage(void) {
+    ubxMessage_t tx_buffer;
+
+    tx_buffer.payload.cfg_nav5x.version = 0x0002;
+
+    tx_buffer.payload.cfg_nav5x.mask1 = 0x4000;
+    tx_buffer.payload.cfg_nav5x.mask2 = 0x0;
+    tx_buffer.payload.cfg_nav5x.reserved0[0] = 0;
+    tx_buffer.payload.cfg_nav5x.reserved0[1] = 0;
+    tx_buffer.payload.cfg_nav5x.minSVs = 0;
+    tx_buffer.payload.cfg_nav5x.maxSVs = 0;
+    tx_buffer.payload.cfg_nav5x.minCNO = 0;
+    tx_buffer.payload.cfg_nav5x.reserved1 = 0;
+    tx_buffer.payload.cfg_nav5x.iniFix3D = 0;
+    tx_buffer.payload.cfg_nav5x.reserved2[0] = 0;
+    tx_buffer.payload.cfg_nav5x.reserved2[1] = 0;
+    tx_buffer.payload.cfg_nav5x.ackAiding = 0;
+    tx_buffer.payload.cfg_nav5x.wknRollover = 0;
+    tx_buffer.payload.cfg_nav5x.sigAttenCompMode = 0;
+    tx_buffer.payload.cfg_nav5x.reserved3 = 0;
```

Comment:
It is probably easier to `memset(&tx_buffer, 0, sizeof(tx_buffer));` and then ignore reserved fields

Reply:
thats a good idea, i will change it to that idea!